docs/theming-flatpak.rst

```diff
@@ -0,0 +1,56 @@
+Theming Flatpak
+===============
+
+Flatpak apps default to Adwaita as a theme, as they cannot use the system theme. This happens because all flatpaks are sandboxed and do not have the ability to read data files or libraries in ``/usr`` (where system themes are typically located). This is done so that the format is portable across a wide range of distributions in a reproducible environment, but has the downside of not being able to pick up themes. The solution to this was to package GTK themes as Flatpaks, as relying upon the host to have the correct version for every flatpak defeats the portability benefits it provides. These themes are provided as extensions to the Gnome runtime.
```

Comment:
Gtk2 and Gtk3 applications default to Adwaita. Qt does not.

Reply:
Flatpak applications in Qt then default to another theme or just show up with no theme at all, and look all ugly? Because I used to have Zukitwo just in my .themes folder, and when I first used Flatpaks, they looked really ugly, and I initially thought there was a problem with the Flatpaks themselves, and then tried another system theme and they looked fine (albeit Adawita-y), so I installed Zukitwo systemwide via a file from a PPA. (Now that I think of it, I should add that element too, in case this happens to someone else...)

Reply:
The default Qt theme is better than say Gtk2's old win95 default but it isn't "pretty"